docs/pages/reference/cli.mdx

```diff
@@ -53,8 +51,14 @@ For more information on subcommands when working with the `teleport` cli, use th
 
 ### teleport start
 
-The `teleport start` command includes a large number of optional configuration flags which change the behavior of the default command to support a wider spectrum of Teleport services.
-We recommend using the [Teleport configuration file](config.mdx) in all cases where possible.
+The `teleport start` command includes a large number of optional configuration flags.
+
+<Admonition title="Configuration Flags" type="tip">
+
+While configuration flags for `teleport start` can be used to set parameters for Teleport's configuration,
```

Comment:
I think we should either move the "The `teleport start` command..." sentence into this Admonition or turn this Admonition into body text, since I don't think the first sentence makes sense outside of the recommendation to use a configuration file. Otherwise, it just sounds like we're passing judgment on the number of optional configuration flags.